### PR TITLE
Docs for Meansible

### PR DESCRIPTION
**Pull request by ghost**
Tell me if any changes are necessary

### CODE REVIEW

File: docs/meansible.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#### Name
+Give your base Box a name, Ya know, to make it yours
+#### IP
+Give your VM instance an IP. for use with SSH, access, etc etc.
+#### MEMORY
+what amount of memory is your VM going to consume? We recommend an amount below half your System Memory.
+####Shared Folder
+Which is the folder you want to sync between your VM and your Local Machine?
+### NODE JS
Select how you would like to install NODE, and which version
```
Comment (mahasooq) [inline docs/meansible.md]: Typo. It should be  Node.js  not 'NODE JS' or 'NODE'

File: docs/meansible.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+#### Chooose your Db
+If you want to install MONGO DB , Check the box.
```
Comment (mahasooq) [inline docs/meansible.md]: MongoDB

**Comment by mahasooq** [comment]
Some typo here and there. Can you please proofread it once again.

**Comment by ghost** [comment]
Should we add a paragraph about ansible too?

**Pull request merged**